src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__seat_tracker_snapshot.sql

```diff
@@ -1,17 +1,36 @@
+with
+    date_spine as (
+        {%- set end_date = (
+            "cast('" + var("current_fiscal_year")
+            | string + "-06-30' as date)"
+        ) -%}
+
+        {{-
+            dbt_utils.date_spine(
+                start_date="cast('2023-08-04' as date)",
+                end_date=end_date,
+                datepart="week",
+            )
+        -}}
+    )
```

Comment:
hey @cbini, not sure why i'm getting an error here but I can't get trunk to be happy

Reply:
I think `date_spine` returns a CTE. does this run in dev?

Anyway, use [generate_date_array](https://cloud.google.com/bigquery/docs/reference/standard-sql/array_functions#generate_date_array)

Reply:
is there a way to set the end date as "today"?